### DISCUSSION TITLE
Hide call argument name inlay hints when parameter and argument name are identical

### DISCUSSION THREAD

**Discussion by disrupted** [microsoft/pylance-release#4834] [Enhancement]
Currently, `callArgumentNames` inlay hints are shown for parameters, even when the actual parameter name is already identical to the variable being passed as argument.

Here's an example
![yield response.follow('url='url, 'callback='self.parse_application)](https://github.com/microsoft/pylance-release/assets/4771462/bc373ec0-551b-43c3-a31c-5206f26c4ee0)

IMO the inlay hint for the `url` parameter is unnecessary, because the argument being passed already has the exact same name. I believe it would be more concise and read nicer if only the inlay hint for the `callback` parameter was shown.

**Comment by debonte** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
